index.html

```diff
@@ -2,377 +2,405 @@
 <html lang="en">
 
 <head>
-    <meta charset="UTF-8">
-    <meta http-equiv="X-UA-Compatible" content="IE=edge">
-    <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Portoflio</title>
-    <link rel="stylesheet" href="styles.css">
-    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Portoflio</title>
+  <link rel="stylesheet" href="styles.css">
+  <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
 </head>
 
 <body>
-    <main class="toolbar">
-        <div class="nav-menu">
-            <a href="#">Welcome</a>
-            <div class="link-list">
-                <a href="#">project</a>
-                <a href="#">About</a>
-                <a href="#">Information</a>
+  <main class="toolbar">
+    <div class="nav-menu">
+      <h4 class="welcome"><a href="#">Welcome</a></h4>
+      <ul class="menu-item">
+        <li class="menu-lists"><a href="#">Projects </a></li>
+        <li class="menu-lists"><a href="#">About</a></li>
+        <li class="menu-lists"><a href="#"> Contact</a></li>
+      </ul>
+      <a href="#" class="menu-icon">
+        <i class="fa fa-bars"></i></a>
+    </div>
+
+    <section class="headline">
+      <div class="primary-text">
+        <h1>Hey there, I'm Addis</h1>
+        <h4>I'm software Engineer</h4>
+      </div>
+      <div class="supporting-text">
+        <p>I can help you build a product , feature or website Look through some of my work and experience! If you like what you
+        see and have a project you need coded, don’t hestiate to contact me.
+        </p>
+      </div>
+      <div class="social-media">
+        <ul class="social-nav">
+          <li><a href="http://linkedin.com/AddisTsega"><i class="fa fa-linkedin"></i></a></li>
+          <li><a href="http://linkedin.com/AdaTsega"><i class="fa fa-twitter"></i></a></li>
+          <li><a href="http://linkedin.com/Aseadd"><i class="fa fa-github"></i></a></li>
+          <li><a href="http://linkedin.com/Addis-Tsega"><i class="fa fa-facebook"></i></a></li>
+        </ul>
+      </div>
+    </section>
+    <section class="recent-works">
+      <div class="project-text">
+        <h1 class="projects">
+          My Recent Works
+        </h1>
+        <h4 class="indicator">
+        </h4>
+      </div>
+      <div class="grid-work-row1">
+        <div class="card-enable1">
+          <div class="img-placeholder1">
+            <div class="project-text">
+              <div class="project-title">
+                <h3 class="project-name">Multi-Post Stories Gain+Glory</h3>
+                <ul class="tags">
```

Comment:
- [ ] The website has got many things that need to be changed to look like the Figma design. Please make sure it resembles the Figma design. 

<p align="center">&nbsp;
 
| Figma design |  your project |
| ---------- | ----------- |
 | ![Template 3-main](https://user-images.githubusercontent.com/12524453/178063534-683db59a-3b61-4125-b457-300d219865a7.png) | ![Screen Shot 2022-07-08 at 23 05 08](https://user-images.githubusercontent.com/12524453/178063136-21a7f858-e3fb-4e72-b1e5-b40de30f08c1.png) |
</p>

Reply:
- Layout, font-size and type is updated and is sililar with figma